parm/land/prep/prep_gts.yaml

```diff
@@ -3,7 +3,11 @@ gtsbufr:
     - $(DATA)/obs
   copy:
     - [$(COM_OBS)/$(OPREFIX)adpsfc.tm00.bufr_d, $(DATA)/obs/]
+    - [$(COM_OBS)/$(OPREFIX)snocvr.tm00.bufr_d, $(DATA)/obs/]
```

Comment:
can you also replace the `$( )` with `{{ }}` in this yaml?

Reply:
Made changes.